meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/supervisor.conf

```diff
@@ -1,3 +1,4 @@
+# This file represents the last known version of the supervisor
```

Comment:
It would probably make sense to remove stable parts like LED_FILE and move those in another configuration file (eg supervisor-setup.conf). In this way we will keep here the bits that are updated (or may be updated) with the supervisor updater - writable though a bind mount.